node/resolvers/catalog/searchBreadcrumb.ts

```diff
@@ -115,10 +115,10 @@ export const resolvers = {
       queryUnit,
     }: BreadcrumbParams) => {
       if (index === 0 && isCategoryMap(mapUnit)) {
```

Comment:
why not adding this case to a single if instance ?
```suggestion
      if (index === 0 && (isCategoryMap(mapUnit) || isBrandMap(mapUnit))) {
```